internal/protobuf/stream.proto

```diff
@@ -27,6 +27,7 @@ option (gogoproto.verbose_equal_all) = true;
 message ID {
     bytes public_key = 1;
     string address = 2;
+    bytes id = 3;
```

Comment:
Can you add documentation in the proto file on the fields. People are going to ask what ID.Id is.

Also mark `public_key` with a `deprecated` comment

Reply:
its not exactly deprecated, we still need to use it in the code